source/api/filters.md

```diff
@@ -139,6 +139,7 @@ Or, just pass in a custom filter function:
 ``` html
 <div v-repeat="user in users | filterBy myCustomFilterFunction">
 ```
+Where `myCustomFilterFunction` is regular `vm` method accepting three arguments `user`, `index` and `users` from the example above.
```

Comment:
It probably make for sense to point out here that the method is expected to be a function that you'd pass to `Array.prototype.filter`.